egs/chime4/s5_1ch/local/clean_chime4_format_data.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright 2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
```

Comment:
You can add info for yourselves if you changed it.  And yes, simplify it as
in Yenda's suggestion.


On Wed, Apr 27, 2016 at 4:00 PM, Shinji Watanabe <notifications@github.com>
wrote:

> In egs/chime4/s5_1ch/local/clean_chime4_format_data.sh
> <https://github.com/kaldi-asr/kaldi/pull/730#discussion_r61325560>:
>
> > @@ -0,0 +1,85 @@
> > +#!/bin/bash
> > +
> > +# Copyright 2012  Microsoft Corporation  Johns Hopkins University (Author: Daniel Povey)
>
> These are originally from wsj/s5/local/wsj_format_data.sh (long time ago)
> How should I modify the copyright?
> Also I noticed that wsj/s5/local/wsj_format_data.sh becomes simple by
> using "utils/validate_lang.pl" etc.
> Should I change it like that?
> This is similar to @jtrmal <https://github.com/jtrmal> 's suggestion
> below.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/730/files/1bb204ec1d3cadd1d7760d9c6bb84d2cb5dd3357#r61325560>
>